### DISCUSSION TITLE
TIKTOK & TAC

### DISCUSSION THREAD

**Discussion by juluz75** [AmauriC/tarteaucitron.js#1050] [Q&A]
Salut à tous,

Je cherche à ajouter sur un site, en mode manuel, le service 'TIKTOK' pour une série de vidéos intégrées à différentes pages, un peu à l'image du service 'vidéo' YouTube et des iframes.

Le seul service identifié TikTok est catégorisé 'mesure d'audience', et me demande une valeur tiktokID que j'ai du mal à trouver dans mon  iframe tiktok ou l'url de ma vidéo.

Si quelqu'un a déjà trouvé comment faire je suis preneur, surtout que Tiktok n'est pas un petit réseau social.

Merciiiiii

Juluz

**Comment by AmauriC** [comment]
Salut : )

Quel est le tag html à ajouter sur le site fourni par Tiktok?

**Comment by juluz75** [comment]
Salut Amauri,

Je te prends l'exemple donné par la page dédiée aux dev sur Tiktok.com.

Pour une vidéo embed/intégrée, soit tu as une URL avec à la fin, l'ID 

`https://www.tiktok.com/@scout2015/video/7187525542424448302?is_from_webapp=1&sender_device=pc&web_id=7152889963019765254`

Soit tu disposes d'un script, un peu à l'image de Youtube. Pour cette même vidéo, ils fournissent ce code embed 

`<blockquote class="tiktok-embed" cite="https://www.tiktok.com/@scout2015/video/7187525542424448302" data-video-id="7187525542424448302" style="max-width: 605px;min-width: 325px;" > <section> <a target="_blank" title="@scout2015" href="https://www.tiktok.com/@scout2015?refer=embed">@scout2015</a> I think the 2nd one is WAY better🤭… <a title="dogsoftiktok" target="_blank" href="https://www.tiktok.com/tag/dogsoftiktok?refer=embed">#dogsoftiktok</a> <a title="animals" target="_blank" href="https://www.tiktok.com/tag/animals?refer=embed">#animals</a> <a title="sosatisfying" target="_blank" href="https://www.tiktok.com/tag/sosatisfying?refer=embed">#sosatisfying</a> <a target="_blank" title="♬ original sound - Scout, Suki &#38; Stella" href="https://www.tiktok.com/music/original-sound-7187525543615613738?refer=embed">♬ original sound - Scout, Suki &#38; Stella</a> </section> </blockquote> <script async src="https://www.tiktok.com/embed.js"></script>`

Avec là aussi, la problématique consistant à avoir plusieurs vidéos tiktok à ajouter sur un même site, et donc un service cumulable coté backoffice.

Juluz

**Reply by AmauriC** [reply]
Parfait, merci. J'ai ouvert une issue, j'essai de m'en occuper rapidement ; )
https://github.com/AmauriC/tarteaucitron.js/issues/1052

**Reply by juluz75** [reply]
Merci Amauri.

J'espère que vous pourrez intégrer ce service.

Bon courage.

Julien
